cmd/meroxa/root/apps/deploy.go

```diff
@@ -834,8 +835,22 @@ func (d *Deploy) tearDownExistingResources(ctx context.Context) error {
 	return nil
 }
 
+// validateFlags take care of some possible incompatibilities between flag values.
+func (d *Deploy) validateFlags() error {
+	if d.flags.Spec == "" && d.flags.Environment != "" {
+		return fmt.Errorf(
+			"please run `meroxa apps deploy` with `--spec %s` or `--spec %s` if you want to deploy to an environment",
+			ir.SpecVersion_0_1_1, ir.SpecVersion_0_2_0)
```

Comment:
So this isn't true if the language is ruby, since it automatically uses a spec-ed version

Reply:
@dianadoherty oops!

Reply:
@dianadoherty it should be fixed now, thank you for the call out.